CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_greenhouse v0.7.0
+[PR #25](https://github.com/fivetran/dbt_greenhouse/pull/25) is a breaking change due to [upstream updates](
+https://github.com/fivetran/dbt_greenhouse_source/blob/main/CHANGELOG.md#dbt_greenhouse_source-v070):
+
+- Updated the logic for `stg_greenhouse__tag` and `stg_greenhouse__user` to account for the presence of the singularly or plurally-named titular source tables, tag(s) and user(s).
```

Comment:
Can you include these in a subsection for upstream breaking changes.

Also be sure to add an under the hood section for the addition of the validation tests

Reply:
Updated!